client/src/components/SimpleModal.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+import { Button, Modal } from 'react-bootstrap'
+
+class SimpleModal extends Component {
+    constructor(props){
```

Comment:
I would add here the propTypes. That way you force certain property types and you also document the component better.